backend/comments-api/README.md

```diff
@@ -0,0 +1,47 @@
+# Kink.com Backend Coding Challenge - Comments API
+
+## Goal
+
+At Kink.com, users can comment on individual scenes. Each scene is identified by an unique positive integer known as a "shootId".
+
+Create a REST API with [Node.js](https://nodejs.org/) & [Express](https://expressjs.com/) that can stores and retrieves comments associated with a scene.
```

Comment:
Change `stores and retrieves` to `store and retrieve`

Reply:
Good catch, I fixed the grammar and a couple other things.